tests/test_parallel.py

```diff
@@ -52,3 +56,11 @@ def test_make_parallel_with_even_data():
 
 def test_make_parallel_with_uneven_data():
     run_parallel_test(generate_uneven_data)
+
+
+def test_keras_make_parallel_with_even_data():
```

Comment:
Keras (correctly) won't run here, since CI doesn't have any GPUs (our function doesn't bother checking)
There's no need to test Keras code anyway, but please still keep the `make_parallel` test in case anyone wants to use it.